iati_dashboard/templates/summary_stats.html

```diff
@@ -19,7 +19,7 @@ <h2 class="dashboard-panel-heading__title float-start">Summary Statistics</h2>
 
 
         <div class="dashboard-panel-body">
-            <p>This table assesses IATI publishers by scoring three dimensions: Timeliness, Forward-looking and Comprehensiveness. The methodology is explained below the table and in the related Publisher Statistics pages. In summary:</p>
+            <p>This table assesses IATI reporting orgs by scoring three dimensions: Timeliness, Forward-looking and Comprehensiveness. The methodology is explained below the table and in the related Reporting Org Statistics pages. In summary:</p>
```

Comment:
I think this should read "Publishing Statistics pages" (ie. it was wrong before)

Reply:
Fixed